core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/scheduling/TestScheduleManager.java

```diff
@@ -241,6 +239,25 @@ public void testDescheduleRunningJob() throws InterruptedException
         assertThat(myScheduler.getQueueSize()).isEqualTo(0);
     }
 
+    @Test
+    public void testGetCurrentJobStatus() throws InterruptedException
+    {
+        CountDownLatch latch = new CountDownLatch(1);
+        UUID jobId = UUID.randomUUID();
+        ScheduledJob testJob = new TestScheduledJob(
+                new ScheduledJob.ConfigurationBuilder()
+                        .withPriority(ScheduledJob.Priority.LOW)
+                        .withRunInterval(1, TimeUnit.SECONDS)
+                        .build(),
+                jobId,
+                latch);
+        myScheduler.schedule(testJob);
+        new Thread(() -> myScheduler.run()).start();
+        Thread.sleep(100);
+        assertThat(myScheduler.getCurrentJobStatus()).contains("Running Job - ID: " + jobId.toString());
```

Comment:
Why not test the complete string (e.g. "... Status: Running")?

Reply:
Done